api/build.gradle

```diff
@@ -20,15 +27,25 @@ repositories {
 }
 
 dependencies {
-	// implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation('org.postgresql:postgresql')
 	developmentOnly 'org.springframework.boot:spring-boot-devtools'
 	testImplementation('org.springframework.boot:spring-boot-starter-test') {
 		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
 	}
 	testImplementation 'org.springframework.security:spring-security-test'
+	compile 'org.flywaydb:flyway-core:6.2.3'
+	testCompile("org.flywaydb.flyway-test-extensions:flyway-spring-test:6.1.0")
+	testRuntime('org.flywaydb:flyway-core:6.2.3')
+}
+
+
+flyway {
+    url = 'jdbc:h2://database/postgres'
+    user = 'postgres'
```

Comment:
This is fine for now, but we should keep in mind that we'll want a safer way to do this later.